### ISSUE TITLE
ERROR! the role 'passstore/ansible_inventory' was not found

### ISSUE THREAD

**Issue by cmoulliard** [snowdrop/k8s-infra#359]
## Issue

The execution of the command `ansible-playbook playbook/openstack/openstack_vm_create_passwordstore.yml` generates this error : 

```

[WARNING]: No inventory was parsed, only implicit localhost is available
[WARNING]: provided hosts list is empty, only localhost is available. Note that the implicit localhost does not match 'all'

PLAY [Init controller] *******************************************************************************************************************************************************************************************************

TASK [Validate required variables] *******************************************************************************************************************************************************************************************
ok: [localhost] => {
    "changed": false,
    "msg": "All assertions passed"
}

TASK [Pull pass git database] ************************************************************************************************************************************************************************************************
changed: [localhost]

TASK [Import generic key] ****************************************************************************************************************************************************************************************************
ERROR! the role 'passstore/ansible_inventory' was not found in /Users/cmoullia/code/snowdrop/k8s-infra/ansible/playbook/openstack/../passstore/roles:/Users/cmoullia/.ansible/roles:/usr/share/ansible/roles:/etc/ansible/roles:/Users/cmoullia/code/snowdrop/k8s-infra/ansible/playbook/openstack/../passstore

The error appears to be in '/Users/cmoullia/code/snowdrop/k8s-infra/ansible/playbook/passstore/passstore_controller_init.yml': line 34, column 15, but may
be elsewhere in the file depending on the exact syntax problem.

The offending line appears to be:

      include_role:
        name: "passstore/ansible_inventory"
              ^ here

PLAY RECAP *******************************************************************************************************************************************************************************************************************
localhost                  : ok=2    changed=1    unreachable=0    failed=0    skipped=0    rescued=0    ignored=0   


```

## How to reproduce 

```
cd $HOME/path/to/k8s-infra/ansible
pass-team (= script used to use password snowdrop team)
source ~/.pyvirt/bin/activate
VM_NAME=dummy-t01
ansible-playbook playbook/openstack/openstack_vm_create_passwordstore.yml -e k8s_type=masters -e k8s_version=123 -e '{"openstack": {"vm": {"network": "provider_net_shared","image": "Fedora-Cloud-Base-35", "flavor": "m1.medium"}}}' -e key_name=test-adm-key -e vm_name=${VM_NAME}
```

**Comment by jacobdotcosta** [comment]
Could you execute the command from `$HOME/path/to/k8s-infra/` ? @cmoulliard

**Comment by cmoulliard** [comment]
```
pwd
/Users/cmoullia/code/snowdrop/k8s-infra

ansible-playbook playbook/openstack/openstack_vm_create_passwordstore.yml -e k8s_type=masters -e k8s_version=123 -e '{"openstack": {"vm": {"network": "provider_net_shared","image": "Fedora-Cloud-Base-35", "flavor": "m1.medium"}}}' -e key_name=test-adm-key -e vm_name=${VM_NAME}

-->

[WARNING]: log file at /opt/log/ansible.log is not writeable and we cannot create it, aborting

[DEPRECATION WARNING]: DEFAULT_GATHER_TIMEOUT option, the module_defaults keyword is a more generic version and can apply to all calls to the M(ansible.builtin.gather_facts) or M(ansible.builtin.setup) actions, use 
module_defaults instead. This feature will be removed from ansible-core in version 2.18. Deprecation warnings can be disabled by setting deprecation_warnings=False in ansible.cfg.
ERROR! the playbook: playbook/openstack/openstack_vm_create_passwordstore.yml could not be found
```

**Comment by jacobdotcosta** [comment]
The documentation was missing the `ansible` folder. Fixed on PR #356 .
